src/heads-up/components/properties-item.vue

```diff
@@ -35,13 +38,17 @@
       <span
         v-if="!valueWithExceededLength"
         class="properties-item__description-text"
-        :class="{ 'properties-item__strike': errors && !valueWithExceededLength }"
+        :class="{
+          'properties-item__strike': errors && !valueWithExceededLength
+        }"
       >{{ value }}</span>
       <span v-if="valueWithExceededLength">{{ valueMinusExceededLength }}</span>
       <span
         v-if="valueWithExceededLength"
         class="properties-item__strike"
-      >{{ valueWithExceededLength }}</span>
+      >{{
```

Comment:
I found the previous version (on one line better readable)

Reply:
This was formatted on save in my code editor, had to disable it to fix it